### DISCUSSION TITLE
Can you consider adding more keyboard code event ?

### DISCUSSION THREAD

**Discussion by joffreypersia** [alpinejs/alpine#3957] [4. General]
Hi, 

Keyboard events like "backspace" or letters can be interesting to listen to because we are building an app with many shortcuts. 

I would love to be able to use something like 
```
@keyup.meta.N="createNewFile()"
@keyup.meta.Backspace="resetForm()"
```

I built it with vanilla JS today, but adding support to regular naming keys would be excellent. 

Thank you!

**Comment by joshhanley** [comment]
@joffreypersia they should already work. See https://alpinejs.dev/directives/on#keyboard-events

**Comment by joffreypersia** [comment]
You're right @joshhanley ! Sorry I missed the line "You can directly use any valid key names exposed via [KeyboardEvent.key](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key/Key_Values) as modifiers by converting them to kebab-case."
